public/src/main/java/com/revenuecat/purchases/errors.kt

```diff
@@ -24,7 +24,7 @@ class PurchasesError(
 enum class PurchasesErrorCode(val code: Int, val description: String) {
     UnknownError(0, "Unknown error."),
     PurchaseCancelledError(1, "Purchase was cancelled."),
-    StoreProblemError(2, "There was a problem with the Play Store."),
+    StoreProblemError(2, "There was a problem with the store."),
```

Comment:
Oh this has been in my mind forever and forgot to change it. Thanks for doing it.